pkg/rke/k8s_rke_system_images.go

```diff
@@ -8858,6 +8858,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/mirrored-pause:3.7",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.21.1",
 		},
+		// Enabled out of band post v2.7.5
+		"v1.24.15-rancher1-1": {
+			Etcd:                      "rancher/mirrored-coreos-etcd:v3.5.4",
+			Kubernetes:                "rancher/hyperkube:v1.24.15-rancher1",
+			Alpine:                    "rancher/rke-tools:v0.1.88",
+			NginxProxy:                "rancher/rke-tools:v0.1.88",
+			CertDownloader:            "rancher/rke-tools:v0.1.88",
+			KubernetesServicesSidecar: "rancher/rke-tools:v0.1.88",
+			KubeDNS:                   "rancher/mirrored-k8s-dns-kube-dns:1.21.1",
+			DNSmasq:                   "rancher/mirrored-k8s-dns-dnsmasq-nanny:1.21.1",
+			KubeDNSSidecar:            "rancher/mirrored-k8s-dns-sidecar:1.21.1",
+			KubeDNSAutoscaler:         "rancher/mirrored-cluster-proportional-autoscaler:1.8.5",
+			Flannel:                   "rancher/mirrored-coreos-flannel:v0.15.1",
+			FlannelCNI:                "rancher/flannel-cni:v0.3.0-rancher6",
+			CalicoNode:                "rancher/mirrored-calico-node:v3.22.5",
+			CalicoCNI:                 "rancher/calico-cni:v3.22.5-rancher1",
+			CalicoControllers:         "rancher/mirrored-calico-kube-controllers:v3.22.5",
+			CalicoCtl:                 "rancher/mirrored-calico-ctl:v3.22.5",
+			CalicoFlexVol:             "rancher/mirrored-calico-pod2daemon-flexvol:v3.22.5",
+			CanalNode:                 "rancher/mirrored-calico-node:v3.22.5",
+			CanalCNI:                  "rancher/calico-cni:v3.22.5-rancher1",
+			CanalControllers:          "rancher/mirrored-calico-kube-controllers:v3.22.5",
+			CanalFlannel:              "rancher/mirrored-flannelcni-flannel:v0.17.0",
+			CanalFlexVol:              "rancher/mirrored-calico-pod2daemon-flexvol:v3.22.5",
+			WeaveNode:                 "weaveworks/weave-kube:2.8.1",
+			WeaveCNI:                  "weaveworks/weave-npc:2.8.1",
+			AciCniDeployContainer:     "noiro/cnideploy:5.2.3.6.1d150da",
+			AciHostContainer:          "noiro/aci-containers-host:5.2.3.6.1d150da",
+			AciOpflexContainer:        "noiro/opflex:5.2.3.6.1d150da",
+			AciMcastContainer:         "noiro/opflex:5.2.3.6.1d150da",
+			AciOpenvSwitchContainer:   "noiro/openvswitch:5.2.3.6.1d150da",
+			AciControllerContainer:    "noiro/aci-containers-controller:5.2.3.6.1d150da",
+			AciGbpServerContainer:     "noiro/gbp-server:5.2.3.6.1d150da",
+			AciOpflexServerContainer:  "noiro/opflex-server:5.2.3.6.1d150da",
+			PodInfraContainer:         "rancher/mirrored-pause:3.7",
```

Comment:
All those tags are older than the ones used in `v1.24.13-rancher2-2`. 
It looks like a mistaken to me unless there is a reason I am unware of.